.github/workflows/tests_linux.yml

```
@@ -207,6 +208,47 @@ jobs:
               -DENABLE_BLAS=ON \
               -DPL_BACKEND=${{ matrix.pl_backend }} \
               -DBUILD_TESTS=ON \
+              -DENABLE_LAPACK=ON \
+              -DCMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION) \
+              -DENABLE_COVERAGE=ON
+            cmake --build ./Build
+            cd ./Build
+            mkdir -p ./tests/results
+            for file in *runner ; do ./$file --order lex --reporter junit --out ./tests/results/report_$file.xml; done;
+            lcov --directory . -b ../pennylane_lightning/core/src --capture --output-file coverage.info
+            lcov --remove coverage.info '/usr/*' --output-file coverage.info
+            mv coverage.info coverage-${{ github.job }}-${{ matrix.pl_backend }}.info
+      
+  cpptestswithOpenBLASNoLAPACK:
+    if: ${{ !contains(fromJSON('["schedule", "workflow_dispatch"]'), github.event_name) }}
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        pl_backend: ["lightning_qubit"]
+    timeout-minutes: 30
+    name: C++ tests (OpenBLAS)
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/setup-python@v4
+        name: Install Python
+        with:
+          python-version: '3.9'
+
+      - name: Checkout PennyLane-Lightning
+        uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get -y -q install cmake gcc-$GCC_VERSION  g++-$GCC_VERSION liblapack-dev libopenblas-dev ninja-build gcovr lcov
+
+      - name: Build and run unit tests
+        run: |
+            cmake . -BBuild -G Ninja \
+              -DCMAKE_BUILD_TYPE=Debug \
+              -DENABLE_PYTHON=OFF \
+              -DPL_BACKEND=${{ matrix.pl_backend }} \
+              -DBUILD_TESTS=ON \
+              -DENABLE_LAPACK=ON \
```

to imply with the name of the job: `cpptestswithOpenBLASNoLAPACK`?
```suggestion
              -DENABLE_LAPACK=OFF \
```

---

Nice spot! Thanks